feat/src/main/java/com/example/feat/document/DocumentScreen.kt

```diff
@@ -0,0 +1,185 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.example.core.components.Skeleton
+import com.example.core.components.VerticalList
+import com.example.core.lib.constants.DesignSystem
+import com.example.feat.R
+import com.example.feat.document.components.TimeRichText
+import com.example.feat.document.components.Timer
+import com.example.feat.document.components.TutorCardView
+import com.example.feat.document.controller.DocumentUiEvent
+import com.example.feat.document.controller.DocumentViewModel
+
+@Composable
+fun DocumentScreen(
+    documentViewModel: DocumentViewModel = hiltViewModel()
+) {
+    val uiState by documentViewModel.uiState.collectAsState()
+    val configuration = LocalConfiguration.current
+    val screenWidth = configuration.screenWidthDp
+
+    Scaffold(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = Color.White.copy(alpha = 0.5f)),
+        topBar = {
+            Row(
+                modifier = Modifier
+                    .padding(vertical = 15.dp, horizontal = 20.dp)
+                    .fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween
+            ) {
+                Row {
+                    Image(
+                        painterResource(R.drawable.ic_logo),
+                        contentDescription = "",
+                        contentScale = ContentScale.Crop,
+                        modifier = Modifier.size(30.dp),
+                    )
+                    Spacer(modifier = Modifier.width(5.dp))
+                    Text(
+                        text = " Tutor",
+                        style = DesignSystem.TITLE_LARGE_STYLE.copy(
+                            color = colorResource(R.color.primaryColor),
+                            fontSize = 24.sp,
+                        ),
+                    )
+                }
+                Row {
+                    Icon(
+                        Icons.Default.Search,
+                        contentDescription = "Search",
+                        modifier = Modifier.size(24.dp),
+                        tint = Color.Black.copy(alpha = 0.7f)
+                    )
+                    Spacer(modifier = Modifier.width(20.dp))
+                    Icon(
+                        Icons.Default.FavoriteBorder,
+                        contentDescription = "Search",
```

Comment:
The content description for the Favorite icon is incorrect. It should not be 'Search'.

```diff
-                        contentDescription = "Search",
+                        contentDescription = "Favorite",
```